common-api/src/main/java/org/pmiops/workbench/cdr/CdrDbConfig.java

```diff
@@ -74,9 +74,7 @@ public CdrDataSource(CdrVersionDao cdrVersionDao,
           cdrVersionId = cdrVersion.getCdrVersionId();
         }
 
-        // TODO: Reconcile this with the public-api DB configs; this should
-        // share code and use the same approach, but attach to the public DBs.
-        String dbName = cdrVersion.getCdrDbName();
+        String dbName = "public".equals(dbUser) ? cdrVersion.getPublicDbName() : cdrVersion.getCdrDbName();
```

Comment:
NIT: our dbUser always has to be named public. That is fine . Too bad there isn't another property on dataSource that says if it's public or private. No biggie.

Reply:
I could make this a config setting if you prefer. But this seemed OK for now; I don't have any reason to expect our DB user to change.

Reply:
Some kind of injected confg property does seem like it would be safer if we already have something. I suppose even if this logic breaks the public user will fail hard trying to access the private CDR index

Reply:
Correct -- only "workbench" has access to the CDR DB. So until I added this check, attempts to access CDR DB info as the public DB user failed. 

We don't already have a config setting for this, but I'm happy to add one if you think it's worthwhile.